patches/chromium/put_back_deleted_colors_for_autofill.patch

```diff
@@ -94,7 +94,7 @@ index f1330721a711c61211ea136286ee884bc00e1975..e479fc84f7013984fc53d18cad8812fb
      kColorId_ThrobberSpinningColor,
      kColorId_ThrobberWaitingColor,
 diff --git a/ui/native_theme/native_theme_win.cc b/ui/native_theme/native_theme_win.cc
-index 5b1804538c2e78af23c797004f250369580b4178..97e72bc38a7231ce295ad943e977c69586cc6e07 100644
+index 5b1804538c2e..97e72bc38a72 100644
 --- a/ui/native_theme/native_theme_win.cc
 +++ b/ui/native_theme/native_theme_win.cc
 @@ -673,6 +673,17 @@ SkColor NativeThemeWin::GetSystemColor(ColorId color_id,
```

Comment:
Please revert these unrelated changes.

Reply:
Sure, I will drop that commit. (Would you accept a separate PR to fix the "trailing whitespace" warning that gets produced without it?)

Reply:
sure :)